.github/workflows/ci.yaml

```diff
@@ -19,11 +19,14 @@ jobs:
 
     - name: Check out code into the Go module directory
       uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+        path: go/src/github.com/pingcap/tidb-operator
```

Comment:
check in GOPATH directory, because we need to install code-generator binaries in `vendor` directory